### ISSUE TITLE
Add /tree slash command to import project structure

### ISSUE THREAD

**Issue by ChristianPerez34** [zed-industries/zed#17568]
### Summary

Support for importing a project directory/file structure (not file contents) as context into the Assistant.

### Description

Example:
```
/tree zed/crates

crates
├── activity_indicator
│   ├── Cargo.toml
│   ├── LICENSE-GPL -> ../../LICENSE-GPL
│   └── src
│       └── activity_indicator.rs
├── anthropic
│   ├── Cargo.toml
│   ├── LICENSE-GPL -> ../../LICENSE-GPL
│   └── src
...
```

Investigate whether tree structure (like `tree crates |head` output above) or alternate text formatting is better ingested as context.  E.g. without unicode box drawing, etc:

```
crates/
  activity_indicator/
    Cargo.toml
```

**Comment by aretrace** [comment]
A simple `/folder` would suffice for now. 🙏

**Comment by Ph0enixKM** [comment]
You can just use `/file` and then provide the path to a directory.

The name should be changed to something like `/path` - this way people wouldn't get so confused

**Comment by aretrace** [comment]
@Ph0enixKM yeah I just recently realized this, a simple `/file` to `/path` rename suffice for now. 😁

**Comment by notpeter** [comment]
The agent panel supports tool calling which can list structures or directly invoke the `tree` command via terminal.  This issue was filed prior to the existence of the agent panel.  We're not currently prioritizing additional features / slash commands for the Zed Assisstant.

Thanks for reporting.
